charts/fybrik/values.yaml

```diff
@@ -296,14 +327,24 @@ opaServer:
   # the equivalent fields of .Values.global.podSecurityContext.
   # ref: https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#security-context
   podSecurityContext: {}
+  # for tls support add:
```

Comment:
why do you need these lines? 330-332

Reply:
without that opa server failed to bind to port 443

Reply:
I changed opa server port to 8443